container-search/src/main/resources/configdefinitions/container.search.schema-info.def

```diff
@@ -0,0 +1,25 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+namespace=container.search
+
+## The name of this schema
+schema[].name string
+
+## The id of the summary class
```

Comment:
this id isn't used anymore, can be dropped

Reply:
Yes, it is commented out. I'll be adding it back when I get rid of documentdb-config, but for that I need to rewrite the way the summary part of that config is generated as it currently is generated from another config, which is no-no.